### ISSUE TITLE
Showing old Sim data

### ISSUE THREAD

**Issue by shrikantr-iprogrammer** [pocesar/react-native-sim-data#12]
If app is in background or in foreground and I change the Sim card, it keeps showing old Sim's details. I had to kill the app and re-open it to get latest data. Can we pull latest Sim details on the run instead of on App start.

**Comment by pocesar** [comment]
yes, it's a known issue, the code is evaluated once at start, and used as a constant thorough the lifetime of the app

**Comment by shrikantr-iprogrammer** [comment]
Thanks for your reply. Do you have an ETA for it?
